tests/CurrencyConverterTest/Provider/FixerApiTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace CurrencyConverter\Provider;
+
+use CurrencyConverter\Provider\Adapter\AdapterInterface;
+use Mockery as m;
+use CurrencyConverter\Provider\Adapter\CurlAdapter;
+
+class FixerApiTest extends \PHPUnit_Framework_TestCase
+{
+    protected function tearDown()
+    {
+        m::close();
+    }
+
+    /**
+     * test the default adapter is CurlAdapter
+     */
+    public function testDefaultAdapter()
+    {
+        $fixer = new FixerApi();
+        $this->assertInstanceOf(CurlAdapter::class, $fixer->getAdapter());
+    }
+
+    public function testGetRate()
+    {
+        $from = 'EUR';
```

Comment:
If you aren't using it more than once there's no point in creating variables here.